controllers/get-moderator.js

```diff
@@ -0,0 +1,15 @@
+const moderators = require('../models/moderator.model');
+const { adminJwtConf } = require('../config/config');
+const jwt = require('jsonwebtoken');
+
+exports.getModerator = async (req, res) => {
+    try {
+        const accessToken = req.header('Authorization');
+        const { userId } = await jwt.verify(accessToken, adminJwtConf.secret);
+        const moderator = await moderators.findAll({where: {id: userId}});
```

Comment:
All logic must be written in model.

Reply:
I will not do it because it will break the functionality that already exists, and at the moment all the logic in us is written in controllers

Reply:
Not all logic written in controllers, if I am not mistaken even you wrote some in model. I think it is not difficult to make import using destructurization and write logic in model. Moreover, the file where you logic must be written is empty so it wont be necessery to change anything and it is a mistake to write it in controllers. If we code according to MVC we should  follow its rules no matter where it was written earlier because now we know how to do it correct and controllers- mistake.
But  it is your choice.

Reply:
I dont have a choice becouse you dont approve this pr) and we will talk about this little problem tommorow)

Reply:
Is it too difficult?

Reply:
As I said above, let's talk tomorrow about this, and everything will be good :)

Reply:
why can`t you answer now?

Reply:
Because we must decide, replace all logic to models, or replace it to contollers, and maby our teammates will help us, which variant better

Reply:
I do not offer to change previous code I offer to  write correct henceforth.

Reply:
We must decide where we will be write logic in future, and change previous code, because a part of logic was written in controllers and other part was written on models, and as for me, its bad practice. We must choose the correct variant tommorow :)

Reply:
Yes, you told earlier  that you want to rewrite all and to move all from model to controller.So you want to make correct variant incorrect? :) :)

Reply:
Haha) No) you are wrong) i sad we must decide like a team, replace all logic to controllers, 5-6 methods, or replace all logic from controllers to models)

Reply:
Remember, Demo is coming)

Reply:
You want to decide how MVC shoul work? :):)

Reply:
GitHub comments is not the place for such off topic conversations. Please discuss only content of current pull request.

Reply:
We are talking about this pull request ;)

Reply:
Done